README.md

```diff
@@ -1,2 +1,41 @@
-# HW01
-Edit the readme
+### HW01
+
+*** 
+# Olivia Pura
+
+Just a few weeks ago, I graduated from Northwestern University with a degree in Biochemistry. I grew up in **Des Plaines, Illinois** and my family currently lives in **Deer Park, Illinois**... it's safe to say that I'll feel right at home at UC! I'm very much looking forward to starting my first year as a BMB student this fall.
+<br><br>
+Though I'm deep-rooted in Illinois, both of my parents grew up in **Poland**, so my body composition is 50% pierogi and 50% wódka. In all seriousness, I'm so proud of my heritage and am incredibly grateful to be a first-generation American. 
+<br><br>
+
+
+***
+* **My Social Medias**
+  + [My Instagram](http://www.instagram.com/oliviapura/)
+  + [My Dog's Instagram](http://www.instagram.com/bambi.bernedoodle/)
+  + [Facebook](http://www.facebook.com/oliviapura1/)
+<br><br> 
+* **Some Fun Facts**
+  + I **love** Crocs, and own 3 pairs. Sadly, they're not lab-approved :(
+  + My Pokémon card collection is both impressive and concerning in size
+  + Concerts and **music festivals** are my happy place - I listen to all music, but my favorite is **EDM**
+<br><br>
+* **Non-Science Hobbies:**
+  + Fitness
+  + Baking
+  + Bullet Journaling
+  + Pageants (I will actually be competing at Miss USA this year, which I'm very excited for!)
+<br><br>
+
+***
+Hopefully, I'll get to know many of you in real life soon! Mrs. 'Rona has definitely been a lot to swallow for an extrovert like me. To finish, I'll leave y'all with my favorite quote at the moment and a photo of my bernadoodle, Bambi:
+
+<br>
+
+> "To love another person is to see the face of God." <br> *- Victor Hugo*
+
+<br>
+
+![](https://scontent-ort2-1.cdninstagram.com/v/t51.2885-15/e35/p1080x1080/79376578_2826487014063607_8637659416288532766_n.jpg?_nc_ht=scontent-ort2-1.cdninstagram.com&_nc_cat=107&_nc_ohc=Rlzs_GhBnQwAX-NR9Ik&oh=d3e7555401f2d875b149092d6969200b&oe=5F2B7C74)
```

Comment:
Nice way to incorporate the quote and make it stand out. And of course, awesome picture!